cmd/res/devices/hp.w200.yaml.example

```diff
@@ -0,0 +1,10 @@
+# Pre-define Devices
+# DeviceList:
+#   name: "hp-w200-01"
+#   profileName: "HP-Webcam-w200"
+#   description: "HP Webcam w200 - 01"
+#   labels: [ "device-usb-camera-example" ]
+#   Protocols:
+#     USB:
+#       path: "/dev/video0"
+#       autoStreaming: "false"
```

Comment:
These are map keys and should stay how they were in the TOML so they match the constants used in the code
```suggestion
#       Path: "/dev/video0"
#       AutoStreaming: "false"
```